BayPass/Model/Map/Location.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Location.swift
+//  BayPass
+//
+//  Created by 凌脩羽 on 3/28/19.
+//  Copyright © 2019 Tim Roesner. All rights reserved.
+//
+// most code cited from https://medium.com/@kf99916/codable-nsmanagedobject-and-cllocation-in-swift-4-b32f042cb7d3
+
+import CoreLocation
+import UIKit
+
+extension CLLocation: Encodable {
```

Comment:
Is this class used anywhere?

Reply:
This class is a place holder for database input

Reply:
I don't we will need this since we no longer store the location in the tickets.

Reply:
ok, will delete this